src/components/ProgressForm/ProgressForm.tsx

```diff
@@ -0,0 +1,79 @@
+import NavBar from '../navBar';
+import InputWithQuestions from './InputWithQuestions';
+import ProgressHeader from './ProgressHeader';
+import styles from '@/components/ProgressForm/ProgressForm.module.scss';
+
+const questions = [
+    {
+        id: 1,
+        name: 'progress',
+        question: 'Task Progress after the previous update',
+    },
+    {
+        id: 2,
+        name: 'plan',
+        question: 'Planned progress before the next update',
+    },
+    {
+        id: 3,
+        name: 'blockers',
+        question: 'List down any blockers that you have',
+    },
+];
+
+const getFormattedDate = (): string => {
+    const date = new Date();
```

Comment:
pass date as an argument and take this function to the utils folder. also write test for it not done yet. handle edge cases of date passed in different format, or not passed at all

Reply:
Because if we need to format any other date apart from the latest date we can pass that date to it and reuse it.

Reply:
But it is just to get the latest date